benchmark.mjs

```diff
@@ -0,0 +1,37 @@
+/* eslint no-undef: "off" */
+
+import {Buffer} from 'node:buffer';
+import benchmark from 'benchmark';
+import {base64ToString, concatUint8Arrays, hexToUint8Array, isUint8Array, stringToBase64, stringToUint8Array, uint8ArrayToBase64, uint8ArrayToHex, uint8ArrayToString} from './index.js';
+
+const largeUint8Array = new Uint8Array(Array.from({length: 1024}).map((_, i) => i));
```

Comment:
This is not a large array in the sense of real-world use-cases. Make it much larger.

Reply:
It also doesn't test realistic data. I would generate random data with `import {randomBytes} from 'node:crypto';`